cypress/support/commands/dBot.js

```diff
@@ -60,3 +60,17 @@ Cypress.Commands.add('c_openDbotThub', () => {
     .click({ force: true })
   cy.c_loadingCheck()
 })
+
+Cypress.Commands.add('c_deleteStrategy', (isMobile) => {
```

Comment:
This will run as you expect anyway but it's good practice to provide a default value:
```suggestion
Cypress.Commands.add('c_deleteStrategy', (isMobile = false) => {
```